### ISSUE TITLE
Unable to access Presentations(Enter Integrant)

### ISSUE THREAD

**Issue by eastack** [weavejester/integrant#105]
![image](https://github.com/weavejester/integrant/assets/17524858/820aa0f0-5a8c-419a-aef1-af92f1054c13)
![image](https://github.com/weavejester/integrant/assets/17524858/d9728ec3-afe3-4b0c-a5a0-80b3a324b9b3)

**Comment by weavejester** [comment]
Unfortunately I can't do a lot about the video as it was recorded and hosted by Skillsmatter. I do have the slides though, so if the video doesn't come back I can substitute it with a link to the slides in PDF form.

**Comment by frou** [comment]
I archive all the YouTube videos I like and apparently this presentation used to be on YT at https://www.youtube.com/watch?v=Y8mI2OZR3Fg because I have the video file. I can upload it to YouTube again if you like @weavejester

**Comment by weavejester** [comment]
As much as I'd love to say go ahead, I'm unsure of the copyright status of the video. My understanding is that while I have copyright over the slides and my own presentation, the copyright over the video itself belongs to Skillsmatter, or whoever now owns their assets now that they've gone into administration.

Perhaps the thing for me to do is to put up the slides for now and redo the presentation at a later date.

**Comment by jf** [comment]
I think as a fallback that would be great (or some other resource to help folks understand the why).

**Comment by jf** [comment]
@weavejester re https://github.com/weavejester/integrant/issues/69#issuecomment-570995108, was there a new presentation? would really love to be able to see one.

**Comment by weavejester** [comment]
No, I haven't done any other presentations on Integrant. I can try to find my old slides.
